src/lib/components/ui/FileInputField/FileInputField.jsx

```diff
@@ -0,0 +1,167 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import getRootValidationStateClassName from '../../../helpers/getRootValidationStateClassName';
+import { withProviderContext } from '../../../provider';
+import transferProps from '../../../utils/transferProps';
+import withForwardedRef from '../withForwardedRef';
+import styles from './FileInputField.scss';
+
+export const FileInputField = ({
+  changeHandler,
```

Comment:
I thought we want to swich to `on*` naming. I would suggest to use `onChange` instead. @mbohal?

Reply:
This is to be addressed in  https://github.com/react-ui-org/react-ui/issues/218.

I think it better to leave it as is here and then changing everything at the same time. It is a little bit more work, but it will keep thinks consistent and clean.